src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
+    log.info(`Your web extension is ready for being submitted on ` +
```

Comment:
This line currently raises an eslint error (https://travis-ci.org/mozilla/web-ext/jobs/209043848#L1227), the reason is that the first part of the string doesn't contain any interpolated value and the eslint rule enforces a single quote string in this case.